app/helpers/application_helper.rb

```diff
@@ -219,7 +219,7 @@ def authoring_at(created, author)
     return if author.nil?
 
     I18n.t(:'js.label_added_time_by',
-           author: author.name,
+           author: html_escape(author.name),
```

Comment:
💭  It's a good idea to fix this on the display side, but shouldn't we possibly fix this on the saving side as well? For fields like a name, etc there shouldn't be a reason to add any html tags.

Reply:
How would fix it on the saving side? By saving it already escaped? Or stripping the unwanted characters?

Reply:
Yeah, that would need to be discussed. But HTML tags should in my opinion be stripped from names. There's not really a reason to have `<foo>` in a name. But that's just my opinion. But maybe out of scope for this issue here :)